scripts/apps/archive/views/preview.html

```diff
@@ -140,7 +140,7 @@
               
               <ul class="text body-text" ng-if="field.field_type === 'urls'">
                 <li ng-repeat="item in selected.preview.extra[field._id] track by $index">{{item.url}}<br>
-                  <p class="nav-space description-text">{{item.description}}</p>
```

Comment:
You can't, translate dynamic values. This is user-entered content, so you don't need to translate it in the first place.

Reply:
done

Reply:
You don't need to comment "done". I get an email when you push a commit :)